### ISSUE TITLE
Alignment of titles in the (translated) Case Studies section 

### ISSUE THREAD

**Issue by melissawm** [numpy/numpy.org#416]
When the titles for some of the Case Studies are translated (see #55), they may take more than one line of text, which results in the images being misaligned:

![Screenshot_20210419_172809](https://user-images.githubusercontent.com/3949932/115299772-9defa900-a135-11eb-9759-312549cf18e3.png)

**Comment by rgommers** [comment]
Eek. This may be a little fiddly to fix.

**Comment by ErnestoFGonzalez** [comment]
I've been trying to think of a way to do this using Flex, but can't find one. Might work with Grid. 

Are you thinking of aligning the centre of the images plus aligning the bottom of the titles, or something different?

**Comment by melissawm** [comment]
@ErnestoFGonzalez  yeah, I think this is reasonable. I believe all images are the same size, so aligning the text should be enough.
